### ISSUE TITLE
Innersource werkwijze 

### ISSUE THREAD

**Issue by RietveldBLD** [ospo-nl/kennisbank#67]
Toevoegen de Innersource werkwijze

- Readme bestand inhoud toevoegen
- Permissies op repositories
- Beschrijving toevoegen
- Contributing file toevoegen

**Comment by marcvanandel** [comment]
Ja! Goeie!

Ik vind het niet gek om intern na te denken over licentie ... of niet @MauriceHendriks ? Dat hoort bij de technische permissies, denk ik? En bij de contributing guide.

**Comment by MauriceHendriks** [comment]
Waar denk je aan qua licenties in de context van inner source?

**Comment by marcvanandel** [comment]
Nou ja ... in kleine bedrijven is het toch wel ons kent ons ... maar in grote(re) bedrijven is het toch wel verstandig om na te denken over wat inzage van broncode tot gevolg kan hebben. Om duidelijk te zijn over intellectueel eigendom en kennis van de broncode. In elk groot bedrijf worden mensen ingehuurd ... die dan zomaar in de broncode kunnen kijken en eventueel daar uit kunnen kopiëren?

Misschien is het meer een stap richting open source met inner source als tussenstap. Als je vanaf het begin dan meeneemt waar je uiteindelijk terecht wilt komen, dan is een (open source) licentie (keuze) goed om ook direct mee te nemen.

Denk ik ...

**Comment by RietveldBLD** [comment]
Precies. InnerSource kan de stap naar Open Source vergemakkelijken, maar hoeft niet te tot Open Source te komen.

**Comment by marcvanandel** [comment]
Vandaag besproken dat in de licentie pagina al beschreven staat (in de noten) dat licenties alleen een rol spelen in geval van distributie ... wat bij inner source niet van toepassing is. Tegelijk zijn de noten een goede context voor de kennis / inhoud van de licentie pagina. Zouden deze dan niet bovenaan moeten staan en/of die context beter moeten beschrijven? We moeten dat nog 's uitwerken ...

Daarnaast hebben we besproken dat het goed is om een inner source pagina toe te voegen ... al is het maar om te vermelden dat er niets te melden is oid; da's ook kennis 😉

**Comment by RietveldBLD** [comment]
Hoi @evavansloten, ja het blijft een goede opstap op van; 
- intern ontwikkelen 
- intern delen
- intern samenwerken
- extern publiceren
- extern samenwerken
te facilteren.

**Comment by jgroenen** [comment]
Voor mijn gevoel een relevante discussie, al is het maar omdat ook hier weer blijkt dat er best wat onduidelijkheid is over wat de rol van een open source licentie precies is en hoe zich dat verhoudt tot ontsluiting van de broncode naar deze of gene en tot intellectueel eigendom dat ontstaat door bijdrage aan die code base en of dat ligt bij de programmeur of bij diens opdrachtgever of werkgever.

Voor mijn gevoel gaat het bij "inner source" vooral over de tools en technieken, en niet zo zeer over intellectueel eigendom. Dat gevoel wordt in mijn optiek ook bevestigd door het Wiki Lemma (https://en.wikipedia.org/wiki/Inner_source): "InnerSource is the use of open source software development best practices and the establishment of an open source-like culture within organizations[1] for the development of its non-open-source and/or proprietary software. The term was coined by Tim O'Reilly in 2000[2] in his column.[3]"

Om het clean te houden zou ik "inner source" daarom niet vermengen met IP en Licentie discussies. Wat wel kan is een losse discussie over Licenties in het geval van niet-open werken, alhoewel ik niet per se van mening ben dat de OSPO daar het expertisecentrum voor is...

**Comment by Geogoeroe** [comment]
Eens met @jgroenen. Overigens ben ik wel over mijn initiele weerstand qua inner source heen. Met name het verhaal op de Ospology bijeenkomst over toepassing inner soource bij Bosch (Georg Gruetter) gaf wel aan hoe belangrijk het kan zijn. Niet qua licenties, maar wel qua samenwerkingsmodel.

**Comment by RietveldBLD** [comment]
Dank @jgroenen en @Geogoeroe voor de toevoegingen. Ik heb een aantal aanpassingen gedaan.
